addons/account/models/account_move.py

```diff
@@ -3043,6 +3047,31 @@ def _extend_with_attachments(self, attachments, new=False):
 
         return success
 
+    ####################################################
+    # Export Electronic Document
+    ####################################################
+
+    def _action_download_electronic_invoice(self):
+        if not self:
+            return False
+        return {
+            'type': 'ir.actions.act_url',
+            'url': '/account/download_edi_documents?%s' % url_encode({'ids': self.ids}),
+            'target': 'new',
+        }
+
+    def get_edi_doc_attachments_to_export(self):
+        # To override; Returns a list of ir.attachments to export
+        return []
```

Comment:
```suggestion
        # To override; Returns a recordset of ir.attachment to export
        return self.env['ir.attachment']
```
Btw, the docstring should be
```py
""" blablabla

:return blablabla
"""
```